packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -38,27 +98,193 @@ contract FundingRateStore is FundingRateStoreInterface, Testable {
         }
     }
 
+    /**
+     * @notice Returns the timestamp at which the current funding rate was published.
+     * @dev The "current funding rate" is defined as that returned by `getFundingRateForIdentifier(id)`
```

Comment:
nit: missing param natspec

Reply:
Done. FYI I only put natspec comments on the external methods to help reviewers but we should still do a PR cleaning up comments in a subsequent PR